README-zh-CN.md

```diff
@@ -5,34 +5,51 @@
 # OceanBase 开发者中心 (ODC)
 
 OceanBase 开发者中心（以下简称 ODC）是开源的全场景数据库开发和数据管理协同工具，通过协同开发解决数据库的变更风险管控、数据管理和数据安全问题。
+![image](https://github.com/oceanbase/odc/assets/15030999/0ec084a6-25e7-4f1c-8fb8-70e9f33c730e)
```

Comment:
please take care the image size, currently 1680 × 1006, which cost too much network transfer while user load github readme page.
I suggest 50% size, reame page occupty less than 40% screen witdh, 800 witdh will be readable for common scenarios.